### PR TITLE
Tweak permissioned node logging.

### PR DESCRIPTION
**Pull request by joelburget**

### CODE REVIEW

File: p2p/permissions.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func isNodePermissioned(nodename string, currentNode string, datadir string, dir
 		permissonedList = append(permissonedList, v.ID.String())
 	}
 
-	log.Debug("isNodePermissioned", "permisssionedList %v", permissonedList)
+	log.Debug("isNodePermissioned", "permisssionedList", permissonedList)
```
Comment (patrickmn) [inline p2p/permissions.go:27]: permisssionedList typo

Comment (joelburget) [reply p2p/permissions.go:27]: This fixes the typo to use the new log format

**Review by bts — APPROVED** [review]

File: p2p/permissions.go (hunk 1 above)
Comment (patrickmn) [reply p2p/permissions.go:27]: there are 3 s'es in permisssioned

Comment (patrickmn) [reply p2p/permissions.go:27]: also, "permissonedList" is a typo

**Pull request merged**